resources/views/content/scripts.twig

```diff
@@ -37,71 +37,47 @@
-                        </div>
-                    </client-only>
-                </div>
-            </div>
-        </div>
-    </div>
+    <findologic-filter-wrapper
+        v-bind:facets="facets"
+        v-bind:filtersPerRow="{{ config('Findologic.filters_per_row') }}"
+        v-bind:fallbackImageColorFilter="'{{ plugin_path("Findologic") }}/images/no-picture.png'"
+        v-bind:fallbackImageImageFilter="'{{ plugin_path("Findologic") }}/images/no-picture-2.png'"
+        v-bind:currentCategory="{{ currentCategory | json_encode }}"
+        v-bind:showCategoryFilter="{{ showCategoryFilter | json_encode }}"
+        v-bind:showSelectedFiltersCount="{{ config('Findologic.show_selected_filter_count') }}"
+    ></findologic-filter-wrapper>
 </script>
 
 <script type="x/template" data-component="item-filter-tag-list">
-    {{ component( "Findologic::ItemList.Components.Filter.FindologicItemFilterTagList" ) }}
     <client-only>
         <findologic-item-filter-tag-list></findologic-item-filter-tag-list>
     </client-only>
 </script>
 
 <!-- SSR:template(item-search) -->
 <script type="x/template" data-component="item-search">
-    {{ component( "Findologic::ItemList.Components.ItemSearch") }}
-
     <client-only>
         <form method="GET" action="{{ languagePath }}/search" @submit="search()">
-            <div class="container-max">
+            <div class="container-max" :class="{'p-0' : $ceres.isShopBuilder}"> 
                 <div class="position-relative">
                     <div class="d-flex flex-grow-1 position-relative my-2">
                         <input name="query"
                                type="search"
                                class="search-input flex-grow-1 px-3 py-2"
                                ref="searchInput"
-                               {% if minimalSearchTermLength > 0 %}
-                                   minlength="{{ minimalSearchTermLength }}"
-                               {% endif %}
+                                {% if minimalSearchTermLength > 0 %}
+                                    minlength="{{ minimalSearchTermLength }}"
+                                {% endif %}
                                :placeholder="$translate('Ceres::Template.headerSearchPlaceholder')"
                                :aria-label="$translate('Ceres::Template.headerSearchTerm')"
-                               @keyup.down="keydown()"
-                               @keyup.up="keyup()"
-                               @focus="isSearchFocused = true"
-                               @blur="onBlurSearchField($event)">
+                               v-model="searchString" 
+                               @keyup.enter="search()" 
+                               @input="onValueChanged($event.target.value)"
+                               @focus="isSearchFocused = true" 
+                               @blur="onBlurSearchField($event)" 
+                               :autofocus="isShopBuilder" 
```

Comment:
Please remove the autofocus, as our suggest opens up on focus.
We already had issues here, where it opened on every page.